apps/app/src/components/UI/Input.tsx

```diff
@@ -34,21 +34,22 @@ export const Input = forwardRef<HTMLInputElement, Props>(function Input(
               {label}
             </label>
           </div>
-          {type === 'startDate' && (
-            <>
-              <input
-                type="checkbox"
-                style={{ position: 'relative' }}
-                onClick={() => {
-                  if (!props.change) return
-                  props.change()
-                }}
-              />
-              <label style={{ position: 'relative' }}>
-                {t('Multiple Days')}
-              </label>
-            </>
-          )}
+          {type === 'startDate' ||
+            (type === 'endDate' && (
+              <>
+                <input
+                  type="checkbox"
+                  style={{ position: 'relative' }}
+                  onClick={() => {
+                    if (!props.change) return
+                    props.change()
+                  }}
+                />
+                <label style={{ position: 'relative' }}>
+                  {t('Multiple Days')}
```

Comment:
```suggestion
                  {t('Ongoing')}
```

This check should probably disable the `endDate` field and just pass in an empty string, since the user should be able to choose not to add an end date.